pkg/component/kubernetes/apiserver/deployment.go

```diff
@@ -774,9 +774,21 @@ func (k *kubeAPIServer) vpnSeedClientInitContainer() *corev1.Container {
 }
 
 func (k *kubeAPIServer) vpnSeedClientContainer(index int) *corev1.Container {
-	nodes := ""
+	var serviceCIDRs, podCIDRs, nodeCIDRs []string
+	nodeCIDR := ""
```

Comment:
Please do not use `nodeCIDRs` and `nodeCIDR`. It is confusing and got me thinking that some lines are not necessary. Please rename them to be visibly distinct.

Reply:
`nodeCIDR` and `nodeCIDRs` are intentionally chosen to correspond with the environment variables `NODE_NETWORK` and `NODE_NETWORKS`.

Reply:
That is easily confusing. Let‘s get the Feature Gate NewVPN graduated soon so that we can get rid of this.